licenses/PolyForm-Shield-1.0.0.txt

```diff
@@ -161,5 +161,4 @@ indirect.
 software under these terms.
 
 **Use** means anything you do with the software requiring one
-of your licenses.
-
+of your licenses.
```

Comment:
We should remove this file as well.

Reply:
same as https://github.com/harness/uicore/pull/1265#discussion_r1534454586